apps.yaml

```diff
@@ -1,10 +1,15 @@
 # TODO schema for auto completion
 
 apps:
-  - name: "My app101"
-    externalUrl: 'https://my-test-app.app-proxy-poc.sandbox.platform.hmcts.net'
+  - name: "Incident"
+    externalUrl: "https://incidents.hmcts.net"
     logoUrl: https://raw.githubusercontent.com/hmcts/azure-app-proxy/main/logos/incident-bot.png
     appRoleAssignments:
-      - Test app
-    internalUrl: 'https://rees-on-bau101.sandbox.platform.hmcts.net'
-    userAssignmentRequired: true
+      - DTS CFT Developers
+      - DTS SDS Developers
+      - DTS Platform Operations
+    internalUrl: "https://response.platform.hmcts.net/"
```

Comment:
this url is currently on the F5 and will cause double auth, you probably need to setup a new ingress or update the private DNS to point directly to traefik (might be best to just do that)?

Reply:
Raised https://github.com/hmcts/azure-private-dns/pull/607